string_lab.js

```diff
@@ -0,0 +1,197 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+//
+//
+let str = 'Hello Stephanie'
```

Comment:
Would this code work if we change str to "Congratz Stephanie"? Also not what the question was asking to do. It asked for first character to be printed which is "H"